frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -56,6 +57,15 @@ export type K8sResourceCommon = {
   metadata?: ObjectMetadata;
 };
 
+export type K8sResourceKind = K8sResourceCommon & {
```

Comment:
`K8sResourceKind` is deprecated and discouraged. We shouldn't expose this in the SDK.